src/EditorFeatures/VisualBasic/SignatureHelp/InvocationExpressionSignatureHelpProvider.MemberGroup.vb

```diff
@@ -82,6 +82,7 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.SignatureHelp
             Dim format = MinimallyQualifiedWithoutParametersFormat
             format = format.RemoveMemberOptions(SymbolDisplayMemberOptions.IncludeType Or SymbolDisplayMemberOptions.IncludeContainingType)
             format = format.RemoveKindOptions(SymbolDisplayKindOptions.IncludeMemberKeyword)
+            format = format.WithMiscellaneousOptions(format.MiscellaneousOptions And (Not SymbolDisplayMiscellaneousOptions.EscapeKeywordIdentifiers))
```

Comment:
This leads to the question: should `SymbolDisplayFormat` get a `RemoveMiscellaneousOptions()` method?  That would be a public API change.  See [here](http://source.roslyn.io/#Microsoft.CodeAnalysis/SymbolDisplay/SymbolDisplayFormat.cs,f1a74c53f582efcb,references).